tests/reflow.js

```diff
@@ -17,15 +18,13 @@ export default {
 		}
 
 		// TODO: Use adoptCSS when https://github.com/LeaVerou/style-observer/issues/74 is fixed
-		iframe.contentDocument.head.insertAdjacentHTML(
-			"beforeend",
-			`<style>
-				@property --registered-custom-property {
-					syntax: "<number>";
-					inherits: true;
-					initial-value: 1;
-				}
-			</style>`,
+		adoptCSS(
+			`@property --registered-custom-property {
+				syntax: "<number>";
+				inherits: true;
+				initial-value: 1;
+			}`,
+			iframe.contentDocument,
```

Comment:
```suggestion
		gentleRegisterProperty("--registered-custom-property", {
			syntax: "<number>",
			inherits: true,
			initialValue: 1,
		}, iframe.contentDocument.defaultView);
```

Reply:
After thinking about this a bit more, I think registering property using `gentleRegisterProperty` makes more sense.

Reply:
And not only that, there's a bug that won't let this test pass anyway (until is fixed): https://github.com/LeaVerou/style-observer/issues/77

Reply:
We can skip it until then. Which reminds me, `skip` should support setting it to a string in addition to `true`, to provide a reason the test is skipped.